src/buttons.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { useState } from "react";
+import swal from "sweetalert";
+
+function TaskForm() {
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+
+  const showAlert = () => {
+    swal({
+      title: "confirm",
+      text: "Are you sure you want to delete this task?",

```

Comment:
warning instead of waring

Reply:
It would be better to use 'Warning'

It looks weird for the user if this is in lowercase

Reply:
It would be better to use 'Warning'

It looks weird for the user if this is in lowercase